_posts/2023-04-05-cockpit-289.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Cockpit 289
+author: garrett
+date: '2023-04-05'
+tags: cockpit, machines, podman
+slug: cockpit-289
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 289, cockpit-machines 287, and cockpit-podman 66:
+
+
+## Metrics: new looks with colorblind friendy colors for graphs
+![228601347-df99586b-3c21-467f-a74d-1c0f020c2afc]({{ site.baseurl }}/images/289-metrics-new-looks-with-colorblind-friendy-colors-for-graphs.png)
+
+## Accounts: Improve password setting
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit/pull/18520>
+{:.note}
+
+There are several improvements to the process of setting password during account creation:
+- The confirmation password is validated and doesn't match, validation is done after each keystroke. This improves user experience, as the user doesn't have to click "Create" whenever they want to know if passwords match.
+- Password validation is now presented as warning, since a weak password is not actually forbidden, but discouraged nevertheless.
+
+![Screenshot from 2023-04-05 12-41-47]({{ site.baseurl }}/images/289-accounts-improve-password-setting.png)
+
+## Machines: Show VM's pending changes
+
+State: **OPEN** (will not be included) @ <https://github.com/cockpit-project/cockpit-machines/pull/998>
+{:.warn}
+
+Add label which indicated whetever VM has pending changes and needs shutdown, and shows what specific configuration changes cause it.
+
+## Machines: if virtualization is disabled in BIOS/EFI we now show message at initial load
```

Comment:
Reviewing heading:

Machines: show an alert when virtualization is disabled in BIOS/EFI